cl/api/templates/judge-api-docs-vlatest.html

```diff
@@ -0,0 +1,180 @@
+{% extends "base.html" %}
+{% load extras %}
+
+{% block title %}Judge APIs for Federal and State Judges and Justices – CourtListener.com{% endblock %}
+{% block og_title %}Judge APIs for Federal and State Judges and Justices – CourtListener.com{% endblock %}
+
+{% block description %}Use these APIs to query and analyze thousands of federal and state judges, including their biographical information, political affiliations, education and employment histories, judgeships, and more.{% endblock %}
+{% block og_description %}Use these APIs to query and analyze thousands of federal and state judges, including their biographical information, political affiliations, education and employment histories, judgeships, and more.{% endblock %}
+
+{% block sidebar %}{% endblock %}
+{% block navbar-p %}active{% endblock %}
+
+
+{% block footer-scripts %}
+  {% include "includes/anchors.html" %}
+{% endblock %}
+
+{% block content %}
+<div class="col-xs-12 hidden-md hidden-lg">
+  <h4 class="v-offset-below-2">
+    <i class="fa fa-arrow-circle-o-left gray"></i>
+    <a href="{% url "rest_docs" version="v3" %}">Back to API Docs</a>
+  </h4>
+</div>
+
+
+<div id="toc-container" class="hidden-xs hidden-sm col-md-3">
+  <div id="toc">
+    <h4 class="v-offset-below-3">
+      <i class="fa fa-arrow-circle-o-left gray"></i>
+      <a href="{% url "rest_docs" version="v3" %}">Back to API Docs</a>
+    </h4>
+    <h3>Table of Contents</h3>
+    <ul>
+      <li><a href="#about">Overview</a></li>
+      <li><a href="apis">The APIs</a></li>
+      <ul>
+        <li><a href="#people-endpoint">Judges &amp; Appointers</a></li>
+        <li><a href="#positions-endpoint">Positions Held</a></li>
+        <li><a href="#affiliations-endpoint">Political Affiliations</a></li>
+        <li><a href="#education-history">Educational Histories</a></li>
```

Comment:
Education Histories -> Educations and Schools ?

Reply:
I sort of deliberately removed schools as an API that we talk much about because it's encompassed by the educational history semantics. For example, nobody wants to look up a particular school. What they want is the educational info of a judge.

Reply:
thats interesting ... because I was using it to group judges by schools the other day, but I admit I may be an edge case

Reply:
Kind of the same thing, I think. Maybe an example of filtering to a particular school would help?

Reply:
I think that makes sense - let me add an example